### PR TITLE
Release preparation

### LINKED ISSUE #80: Fix graphing grouped runs
context menu creates an issue with run-number format of grouped runs.

### PR DESCRIPTION
**Pull request by EthanDevlinSTFC**
Clean and refactor code
Add a series of comments
Fix found bugs (group ordering and titles)
Rename elementss
Remove obsolete code and ui elements

closes #80

### CODE REVIEW

File: backend/nexusInteraction.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def runData(file, fields, run):
 
 
 def runFields(instrument, cycle, runs):
-    runArr = runs.split(";")
+    runArr = runs.split("-")
```
Comment (trisyoungs) [inline backend/nexusInteraction.py]: I can't get the nexus fields when I have more than one run selected - here you changed the delimited from `;` to `-`, but the error message seems to suggest that you're still adding `;` to the URL:
```
Error2: Error transferring http://127.0.0.1:5000/getNexusFields/nimrod/cycle_21_1/72544;72545;72546 - server replied: INTERNAL SERVER ERROR
```

File: frontend/mainwindow.ui [hunk 2]
```diff
@@ -465,21 +311,33 @@
    <property name="text">
     <string>&amp;Find (Ctrl + F)</string>
```
Comment (trisyoungs) [inline frontend/mainwindow.ui]:
```suggestion
    <string>&amp;Find</string>
```

File: frontend/mainwindow.ui [hunk 3]
```diff
@@ -465,21 +311,33 @@
...
    </property>
+   <property name="shortcut">
+    <string>Ctrl+F</string>
+   </property>
   </action>
   <action name="actionSelectNext">
    <property name="text">
     <string>Find Next (F3)</string>
```
Comment (trisyoungs) [inline frontend/mainwindow.ui]:
```suggestion
    <string>Find Next</string>
```

File: frontend/mainwindow.ui [hunk 4]
```diff
@@ -465,21 +311,33 @@
...
    </property>
+   <property name="shortcut">
+    <string>F3</string>
+   </property>
   </action>
   <action name="actionSelectPrevious">
    <property name="text">
     <string>Find Previous (Shift + F3)</string>
```
Comment (trisyoungs) [inline frontend/mainwindow.ui]:
```suggestion
    <string>Find Previous</string>
```

File: frontend/mainwindow.ui [hunk 5]
```diff
@@ -465,21 +311,33 @@
...
    </property>
+   <property name="shortcut">
+    <string>Shift+F3</string>
+   </property>
   </action>
   <action name="actionSelectAll">
    <property name="text">
     <string>Select All Occurances (Ctrl + F3)</string>
```
Comment (trisyoungs) [inline frontend/mainwindow.ui]:
```suggestion
    <string>Select All Occurrences</string>
```

**Review by trisyoungs — CHANGES_REQUESTED** [review]
Good tidy!

Some comments and changes below.

File: backend/nexusInteraction.py (hunk 1 above)
Comment (EthanDevlinSTFC) [reply backend/nexusInteraction.py]: Oops! I know what I've done there

**Comment by EthanDevlinSTFC** [comment]
Fixed a few more issues

**Pull request merged**